internal/connect/command_offset.go

```diff
@@ -0,0 +1,76 @@
+package connect
+
+import (
+	"fmt"
+
+	connectv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect/v1"
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/config"
+	"github.com/confluentinc/cli/v3/pkg/kafka"
+	"github.com/confluentinc/cli/v3/pkg/types"
+)
+
+type offsetCommand struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+func newOffsetCommand(cfg *config.Config, prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "offset",
+		Short: "Manage offsets for managed connectors.",
+	}
+
+	c := &offsetCommand{}
+
+	c.AuthenticatedCLICommand = pcmd.NewAuthenticatedCLICommand(cmd, prerunner)
+	cmd.AddCommand(c.newGetOffsetCommand())
+	cmd.AddCommand(c.newAlterOffsetCommand())
+	cmd.AddCommand(c.newGetAlterOffsetStatusCommand())
```

Comment:
```suggestion
	cmd.AddCommand(c.newAlterCommand())
	cmd.AddCommand(c.newDescribeCommand())
	cmd.AddCommand(c.newStatusCommand())
```

This is to match our usual convention for naming command functions. The change `newGetOffsetCommand` -> `newDescribeCommand` is because of another comment below.